docs/development/repository_structure.md

```diff
@@ -0,0 +1,92 @@
+# Repository Structure
+
+The repository for Tractus-X EDC can be found [here](https://github.com/eclipse-tractusx/tractusx-edc).
+It contains the following components:
+
+## EDC Extensions
+
+The core EDC is extensible by design.
+Tractus-X EDC provides such extensions.
+These extensions and their documentation are available
+[here](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-extensions/README.md).
+
+
+### Business Partner Validation
+
+This extension allows for validation of business partners within the access policy.
+
+### Control Plane EDR APIs
+
+The goal of this extension is to simplify the process of retrieving data out of EDC.
+It returns `EndpointDataReference` object, hiding all the communication details for contract offers,
+contract negotiation, transfer process and retrieving the underlying data through the data-planes.
+
+### Data Plane HTTP OAuth2
```

Comment:
the title of the document is `Repository Structure`, but this extension is not part of the repository, anything that's not part of the repository should not be listed here